factory/parser.ts

```diff
@@ -97,7 +99,8 @@ export function createParser(program: ts.Program, config: Config, augmentor?: Pa
         .addNodeParser(new NeverTypeNodeParser())
         .addNodeParser(new ObjectTypeNodeParser())
         .addNodeParser(new AsExpressionNodeParser(chainNodeParser))
-
+        .addNodeParser(new FunctionParser(chainNodeParser))
+        .addNodeParser(withJsDoc(new ParameterParser(chainNodeParser)))
```

Comment:
Why do you need to add withJsDoc? We don't add it anywhere else.

Reply:
As far as I can tell that's what I had to do for it to pick up the annotated parameters in the outputted schema. Is there any other way to do it?

Reply:
None of the other parsers add it so I think so. Please check.

Reply:
Looking at the list of parsers I think it actually makes sense to keep the `withJsDoc` on `ParameterParser`. Currently `TypeAliasNodeParser`, `EnumNodeParser`, `InterfaceAndClassNodeParser` & `TypeLiteralNodeParser` are wrapped with `withJsDoc`. Similar to these types of node we're not introducing support of another node (`Parameter`) that can be _also_ be annotated with jsdoc similar to the previously mentioned nodes

I've tried to show what I mean in this [ts-ast-viewer snippet](https://ts-ast-viewer.com/#code/PQKhAIAEBMFMGcDGAnAlgB1QewHbgCoCe6s4AggDaoCG84AIrIhdctQC7Z5xJrqe5wIYACh2xUgCVYAMyIlwAXnAAjLFgqxqOESMS547cAFtCAMQCuORALzKAFCPDPwoCDAQoMt8AEkcVDikPF78XEKiLuCGyABc0exoOADmADROLsiysFnWsNDysPHSchIiAJRKAHzgAEQA7gAWHLAAbjm1ukA), if you click between the two jsdoc comments you'll see it highlight "TypeAliasDeclaration" & "Parameter" in the tree view

Reply:
Ahh great. Makes sense. https://ts-ast-viewer.com/ is pretty nice. I've been using https://astexplorer.net/ so far.